### DISCUSSION TITLE
worried

### DISCUSSION THREAD

**Discussion by ReVerseYTReal** [ThioJoe/YT-Spammer-Purge#1143] [Q&A]
just making sure this is THE REAL thiojoe, i dont wanna get hacked!!

**Comment by Lightning11wins** [comment]
In [this video](https://youtu.be/2tRppXW_aKo) on his channel, he links to this github, so there shouldn't be any doubt that github.com/ThioJoe is real.

**Comment by Firecul** [comment]
It is, been here for a long time

**Comment by ReVerseYTReal** [comment]
ok
